_posts/2022-12-21-easily-verify-your-rust-in-ci-with-kani.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Easily verify your Rust in CI with Kani and Github Actions"
+---
+
+Formal verification tools like Kani definitively check that certain classes of bugs will not occur in your program under any circumstance.
+Kani uses mathematical techniques to explore your code over all inputs, meaning that while testing is a great way to catch the *first* bug, formally verifying your code with Kani guarantees that you've caught the *last* bug.
```

Comment:
This is a very strong statement. Even because we don't catch every source of UB. So I would prefer if we tune this down.

Reply:
Toned down